core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -138,6 +142,45 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.addQuestion(QuestionObjectFactory.create(
           lastStateName, stateData.interaction, ''));
       },
+      changeQuestion: function(type, index) {
+        var currentStateName = (
+          index === 0 ? 'Introduction' : 'Question ' + index);
+        var questionCount = data.questionList._questions.length;
+        var answerGroupsLength = (
+          data.questionList._questions[index]._answerGroups.length);
+        var nextStateName = 'Question ' + (index + 1);
+        var currentInteractionId = (
+          data.questionList._questions[index]._interactionId);
+
+        // Update Question Type If interactionId is not same.
+        if(type !== currentInteractionId) {
+          var newAnswerGroups = [];
+
+          switch (type) {
+            case 'MultipleChoiceInput':
+              SimpleEditorShimService.saveInteractionId(
+                currentStateName, DEFAULT_INTERACTION.ID);
+              SimpleEditorShimService.saveCustomizationArgs(
+                currentStateName, DEFAULT_INTERACTION.CUSTOMIZATION_ARGS);
+              newAnswerGroups.push(AnswerGroupObjectFactory.createNew([
+                RuleObjectFactory.createNew('Equals', {
+                  x: 0
+                })
+              ], OutcomeObjectFactory.createEmpty(nextStateName), false));
+              break;
+          }
+          if(answerGroupsLength !== 0 && index !== questionCount - 1) {
+            SimpleEditorShimService.saveAnswerGroups(currentStateName
```

Comment:
If you are writing something in parentheses and it extends more than a line, break after '('. In general we do not break lines before commas.